tests/test_unit_proxy.cpp

```diff
@@ -107,11 +95,7 @@ void test_httpproxy_fromenv(void **unused)
 
 void test_noproxy_fromenv(void **unused)
 {
-    if (sf_getenv("all_proxy") || sf_getenv("https_proxy") ||
-        sf_getenv("http_proxy")) {
-        // skip the test if the test environment uses proxy already
-        return;
-    }
+    SKIP_IF_PROXY_ENV_IS_SET;
```

Comment:
sometimes we use this constant with `;` sometimes not - let's pick one format

Reply:
done